server/src/infra/repositories/person.repository.ts

```diff
@@ -10,6 +11,53 @@ export class PersonRepository implements IPersonRepository {
     @InjectRepository(AssetFaceEntity) private assetFaceRepository: Repository<AssetFaceEntity>,
   ) {}
 
+  async updateAssetsId(primaryPersonId: string, mergePersonId: string, assetIds: string[]): Promise<number> {
+    const result = await this.assetFaceRepository
+      .createQueryBuilder()
+      .update()
+      .set({ personId: primaryPersonId })
+      .where({ assetId: In(assetIds), personId: mergePersonId })
+      .execute();
+
+    return result.affected ?? 0;
+  }
+
+  /**
+   * Find the assets that containers faces from both persons.
+   * Normally, this shouldn't happen, however there are some edge cases, where your photo is taken
+   * with an older version of your self, and the face detection algorithm detects both faces as different.
+   * @param ids Array of personId.
+   * @returns sAn array of assetId that contains faces from both persons.
+   */
+  async getIdenticalAssets(ids: string[]): Promise<string[]> {
```

Comment:
I don't know if there is great value in returning these. We should just delete the duplicates directly.